diff --git a/aerospace/.config/aerospace/aerospace.toml b/aerospace/.config/aerospace/aerospace.toml
index 2645655..905b658 100644
--- a/aerospace/.config/aerospace/aerospace.toml
+++ b/aerospace/.config/aerospace/aerospace.toml
@@ -63,7 +63,7 @@ outer.bottom =     8
 # see https://github.com/nikitabobko/AeroSpace/issues/492
 # notch height is 32, sketchybar is 32 + 4 offset, 4 gap is 4+4=8
 # sketchybar is 32 + 4 offset, 4 gap withought notch is 32+4+4=40
-outer.top =        [{ monitor."^built-in retina display$" = 8 }, 40]
+outer.top =        40
 outer.right =      8
 
 # 'main' binding mode declaration
@@ -229,3 +229,7 @@ if.app-id="com.mitchellh.ghostty"
 run= [
   "layout floating",
 ]
+
+# Intel mac PATH bug, See https://github.com/nikitabobko/AeroSpace/issues/556#issuecomment-2392419701
+[exec.env-vars]
+PATH = '/usr/local/bin:${PATH}'
